### PR TITLE
chore(github): Add Issue & PR template

### PR DESCRIPTION
**Pull request by ephread**
**Checklist for this pull request**

- [x] The new code additions passed the tests (`npm test`).
- [x] The linter ran and found no issues (`npm run-script lint`).

**Description**

I went ahead I reproduced the template above. Feel free to suggest any change, I'm happy to update the PR.

I also created an issue template, let me know what you think!

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE/pull_request.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+**Checklist for this pull request**
+
+<!--
+    Thank you for your contribution! Before submitting this PR, please
+    make sure that:
+-->
+
+- [ ] The new code additions passed the tests (`npm test`).
+- [ ] The linter ran and found no issues (`npm run-script lint`).
```
Comment (y-lohse) [inline .github/PULL_REQUEST_TEMPLATE/pull_request.md:14]: Maybe we can add a ligne here with something like "Running `./node_modules/.bin/tslint -c tslint.json 'src/**/*.ts' --fix` will fix most of the linting problems automatically", or something like that?

Comment (ephread) [reply .github/PULL_REQUEST_TEMPLATE/pull_request.md:14]: Good idea, maybe it's worth adding a new script named: "lint:fix" to wrap this command as part of the PR?

Comment (y-lohse) [reply .github/PULL_REQUEST_TEMPLATE/pull_request.md:14]: Yep, good idea! I just realized `npm run lint -- --fix` also works, but `lint:fix` is probably less confusing than the quadruple dash 😁.

File: package.json [hunk 2]
```diff
@@ -12,6 +12,7 @@
     "test:remap:json": "remap-istanbul --input coverage/coverage-final.json --output coverage/coverage-final.json",
     "build": "tsc -p ./ && rollup -c",
     "lint": "tslint -c tslint.json 'src/**/*.ts'",
+    "lint:fix": "npm run lint -- --fix",
```
Comment (ephread) [inline package.json:15]: Best of both worlds!

**Comment by ephread** [comment]
I replaced `**` by `##`, that's the only other change beyond what we discussed. 🙂

**Pull request merged**